### PR TITLE
WNP 95 for /de/ (Fixes #10800)

### LINKED ISSUE #10800: Whats New Page (de)
## Description

WNP for Fx95 de is Pocket focused. Fx95 releases on Dec 7.

[Design](https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/Firefox-WNP-EU-(Copy-June-2021)?node-id=916%3A2249)

[Copy](https://docs.google.com/document/d/1Ef1UPSKEqxTropynqe_hlEgou56eX3Rd7GSLkceDFSU/edit?usp=sharing)

No split test. No languages for EU besides DE.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds /whatsnew page for Firefox 95 in German.

## Issue / Bugzilla link
#10800

## Testing
http://localhost:8000/de/firefox/95.0/whatsnew/

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for review

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html:45]: Curious: why not use macros here?

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (reemhamz) [inline media/css/firefox/whatsnew/whatsnew-95-de.scss:24]: This class doesn't look like it's being used?

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (reemhamz) [inline media/css/firefox/whatsnew/whatsnew-95-de.scss:63]: I don't think this is used in the DOM either?

**Review by reemhamz — CHANGES_REQUESTED** [review]
Looks really well put together! Just a couple of nits if I'm correct in spotting them. r+wc! 💫

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html:45]: Only because I'm using inline SVG assets here (for better WNP performance), rather than adding additional HTTP requests for the images.

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (alexgibson) [reply media/css/firefox/whatsnew/whatsnew-95-de.scss:24]: Good catch, will remove 👍

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (alexgibson) [reply media/css/firefox/whatsnew/whatsnew-95-de.scss:63]: Will remove 👍

**Comment by alexgibson** [comment]
@reemhamz thanks for the review, updated!

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (maureenlholland) [inline media/css/firefox/whatsnew/whatsnew-95-de.scss:69]: was there a request to change the footer? it seems to be the regular grey for both light and dark themes in the mockups

for light theme, the link styling is a bit off:
- regular links don't get an underline on hover like they do on the dark theme
Screen Shot 2021-11-26 at 11 25 40 AM

- the social icons do get an underline on hover, but I don't think that's the desired look
Screen Shot 2021-11-26 at 11 25 05 AM

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html:47]: Screen Shot 2021-11-26 at 11 13 18 AM

since we're inlining svg but there's no valuable information we need to convey, it's probably worth adding `aria-hidden=true` on the svgs to remove them from the accessibility tree

**Review by maureenlholland — CHANGES_REQUESTED** [review]
good 👀 @reemhamz and great work @alexgibson !

I too only have minor nitpicky things

File: media/css/firefox/whatsnew/whatsnew-95-de.scss
Comment (alexgibson) [reply media/css/firefox/whatsnew/whatsnew-95-de.scss:69]: @maureenlholland this is the standard footer theme we use for WNPs, no new changes added here just reuse of existing code from prior WNPs.

Comment (alexgibson) [reply media/css/firefox/whatsnew/whatsnew-95-de.scss:69]: I will see if we can improve the icon hover styling though 👍

**Comment by alexgibson** [comment]
@maureenlholland updated thanks. I tadded a commit to make the border hover states more consistent between theme styles.

I think I'm going to file an issue for tidying up / improving our footer. This code is copy/pasted between every version of the WNP, which make maintenance a chore. It should really be using a shared stylesheet. I think we could also make it more consistent with the standard footer styles.

**Comment by alexgibson** [comment]
Filed a followup issue to improve the way we use shared footer styles on these pages: https://github.com/mozilla/bedrock/issues/10829

**Review by maureenlholland — APPROVED** [review]
r+ 👍

**Comment by alexgibson** [comment]
@reemhamz anything else you wanted to add here? I think still need your approval before this can merge. Thanks again!

**Review by reemhamz — APPROVED** [review]
Looks good to me! ✨🪐

**Pull request merged**
